tests/mock/zwave.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import MagicMock
+
+from pydispatch import dispatcher
+
+SIGNAL_VALUE_CHANGED = 'mock_value_changed'
+
+
+def value_changed(value):
+    """Fire a value changed."""
+    dispatcher.send(
+        SIGNAL_VALUE_CHANGED,
+        value=value,
+        node=value.node,
+        network=value.node._network
+    )
+
+
+class MockNode(MagicMock):
+    """Mock Z-Wave node."""
+
+    def __init__(self, *,
+                 node_id='567',
+                 name='Mock Node',
+                 manufacturer_id='ABCD',
+                 product_id='123',
+                 command_classes=None):
+        """Initialize a Z-Wave mock node."""
+        super().__init__()
+        self.node_id = node_id
+        self.name = name
+        self.manufacturer_id = manufacturer_id
+        self.product_id = product_id
+        self._command_classes = command_classes or []
+
+    def has_command_class(self, command_class):
+        """Test if mock has a command class."""
+        return command_class in self._command_classes
+
+    def _get_child_mock(self, **kw):
```

Comment:
The default behavior is to create a child mock with an instance of the current class. This is not possible because we overwrite the `__init__` method.